Exercise_C/index.css

```diff
@@ -2,3 +2,31 @@
 This css sheet is linked to index.html
 You can style tags, classes (.) or ids (#) from this file.
 */
+
+
+.container {
+    padding: 10px;
+    border: 0px;
+    margin: 0 auto;
+    display: flex;
+    justify-content: space-between;
+    height: 200px;
+}
+
+.box{
+    height: 100px;
+    width: 100px;
+}
+
+#blue{
+    background-color:blue;
+}
+
+#green{
+    background-color: green;
+}
+
+#red{
+    background-color: red;
+    align-self: flex-end;
+}
```

Comment:
Fantastic use of the align-self, flex-end css! 
Well done for increasing the height of the container to use in conjunction with flex end.